app/templates/ui/app/create/create.controller.spec.js

```diff
@@ -9,10 +9,12 @@
 
     beforeEach(function() {
       bard.appModule('app.create');
-      bard.inject('$controller', '$q', '$rootScope', 'MLRest', '$state', 'userService');
+      bard.inject('$controller', '$q', '$rootScope', 'MLRest', '$state', 'userService',
+        'x2js');
 
       bard.mockService(MLRest, {
-        createDocument: $q.when('/?uri=blah')
+        createDocument: $q.when('/?uri=blah'),
+        updateDocument: $q.when('/?uri=blah')
```

Comment:
Hmm, it is now only testing editing of a document I think. Maybe we need to create two copies of the controller, one for create and one for update, and duplicate the `should add tags` test case, one for create and one for update.. that should suffice..